api/notd/api/api_v1.py

```diff
@@ -88,6 +89,11 @@ async def update_activity_for_all_collections_deferred():
         await notdManager.update_activity_for_all_collections_deferred()
         return UpdateActivityForAllCollectionsDeferredResponse()
 
+    @router.post('/collections/update-attributes-deferred', response_model=UpdateAttributeForAllTokensDeferredResponse)
```

Comment:
this path is wrong. try to think of something better. if you cant i will but you try first